developers/advanced/advanced-pinot-setup.md

```diff
@@ -311,6 +311,33 @@ bin/pinot-admin.sh AddTable \
 {% endtab %}
 {% endtabs %}
 
+### Automatically add an inverted index to your batch table
+
+It is possible to force the creation of an inverted index during segment generation by adding an entry to your [table index config](../../configuration-reference/table.md#table-index-config) in the table configuration file.
+
+This setting works with [batch (offline) tables](#batch-table-creation). This setting is `false` by default but you can set `createInvertedIndexDuringSegmentGeneration` to `true` in your table config, as follows:
+
+```json
+...
+"tableIndexConfig": {
+    ...
+    "createInvertedIndexDuringSegmentGeneration": true,
+    ...
+}
+...
+```
+
+When set to `true`, Pinot creates an inverted index for the columns that you specify in the `invertedIndexColumns` list in the table configuration. If `false`, no inverted index is created.
```

Comment:
is this still only for batch tables? if yes, we may want to combine with the first paragraph as an overview for what the whole thing is, give the example, then say, what you must do after to apply the index...